airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -126,6 +135,27 @@ def __init__(
         self._doc_type_map: dict[Any, Any] = {}
         self._doc_type: list[Any] = []
 
+    @staticmethod
+    def format_url(host: str) -> str:
+        """
+        Formats the given host string to ensure it starts with 'http' or 'https'.
```

Comment:
I think we need to rephrase this as we only try prefixing `http` and not `https` if the URL does not have a scheme.

Reply:
changed it: https://github.com/apache/airflow/pull/33281/commits/2984194c7f3be440831c6b92d577271c9027600b